src/main/java/blackjack/domain/participant/Players.java

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.participant;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Players {
+
+    private static final int MIN_PLAYER_SIZE = 1;
+    private static final int MAX_PLAYER_SIZE = 4;
+
+    private final List<Player> players;
+
+    Players(List<Player> players) {
+        this.players = players;
+    }
+
+    static public Players from(List<String> playerNames, List<Integer> bettings) {
+        validateSize(playerNames, bettings);
+
+        List<Player> players = IntStream.range(0, playerNames.size())
+                .mapToObj(i -> new Player(playerNames.get(i), new Betting(bettings.get(i))))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        List<Player> players = IntStream.range(0, playerNames.size())
                .mapToObj(i -> new Player(playerNames.get(i), new Betting(bettings.get(i))))
                .toList();
```
Java 에서 제공하는 문법을 적극 활용해보죠!
`toList()` 와 `.collect(Collectors.toList())` 의 차이를 공부해봐도 좋을 것 같아요.

Reply:
collect(toList()) 는 ArrayList 반환 하여 수정이 가능하고
toList() 는 UnmodifiableList 를 반환하여 수정이 불가능하네요.

상황에 따라 선택하되 가독성 및 수정을 막는 toList가 일반적으로 더 좋아보이네요!